src/modules/helpchan.ts

```diff
@@ -239,6 +246,113 @@ export class HelpChanModule extends Module {
 		}
 	}
 
+	private async addCooldown(member: GuildMember, channel: TextChannel) {
+		await member.roles.add(askCooldownRoleId);
+		const helpUser = new HelpUser();
+		helpUser.userId = member.user.id;
+		helpUser.channelId = channel.id;
+		await helpUser.save();
+	}
+
+	@command()
+	async cooldown(msg: Message, @optional user?: GuildMember) {
+		console.log('Cooldown', msg.content);
+		if (!msg.guild) return;
```

Comment:
Use [CommonInhibitors](https://github.com/cookiecord/cookiecord/blob/master/src/command/inhibitors.ts#L18) and the `!` syntax?